src/ex4/client/src/services/TodoApiServies.js

```diff
@@ -0,0 +1,53 @@
+import axios from "axios";
+
+async function apiGetTodos() {
+  try {
+    const { data } = await axios.get("/item");
+    return data;
+  } catch (error) {
+    throw new Error("Canot connect to server, please try again later");
```

Comment:
if you just catch any error and throw this then you don't have an option to handle specific errors that you might want communicated from the server, in this case you are getting an error and ignoring it, not even logging it